packages/components/src/Card/Card.css

```diff
@@ -56,6 +56,12 @@
 }
 
 .header {
+  display: flex;
+  flex-direction: row;
   padding: var(--card--base-padding);
   background-color: var(--color-surface--background);
 }
+
+.headerButton {
+  margin-left: auto;
```

Comment:
Does this class actually need to exist if this is all it's doing? It seems like we could just add an equivalent `gap` to the `header` class, no?

Reply:
the `margin-left: auto` was really meant for right aligning the button, which is now better handled by `header` class with `justify-content: space-between`, and so the `headerButton` class can be removed.

Reply:
We'll still need the gap, as the title can currently run right into the action before wrapping: 
![image](https://user-images.githubusercontent.com/39704901/217896973-d2ff9962-4340-41cf-8bc2-4d7db9311c70.png)

Reply:
As i was adding `gap`, i realized that the wrapping would vertically stretch out the button. To counteract this, I added `align-content: top`, which raises the question as to whether this should be another prop. This is likely something to consider along with your comments about the size of buttons.  
![image](https://user-images.githubusercontent.com/94410044/217938857-18f15dd0-18c1-494f-83a3-c5d337b1b282.png)